package.json

```diff
@@ -13,11 +13,13 @@
     "bootstrap": "pnpm setup:dev",
     "copy:files": "cpx README.md packages/core",
     "prerelease": "pnpm copy:files",
-    "start:web": "cross-env lerna run --parallel --include-dependencies --scope @notifire/web start:static:build",
-    "start:widget": "cross-env lerna run --parallel --include-dependencies --scope @notifire/widget start:test",
-    "start:ws": "cross-env lerna run --parallel --include-dependencies --scope @notifire/ws start:test",
+    "start:web": "cross-env nx run @notifire/web:start",
+    "start:widget": "cross-env nx run @notifire/widget:start",
```

Comment:
@djabarovgeorge Looks like it was "start:test" before here and that's why the tests are falling right now.

Reply:
Yes not sure from where the change came I will update it back. Thanks for the ping. 🙃